Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackQualityMetricsTests.swift

```diff
@@ -13,6 +13,7 @@ class AVFoundationPlaybackQualityMetricsTests: QuickSpec {
                         let baseObject = BaseObject()
                         var bitrate: Double?
                         let avfoundationPlayback = AVFoundationPlayback(options: [:])
+                        avfoundationPlayback.render()
```

Comment:
I've seen this line being added to a lot of tests, do you think we could create a `beforeEach` closure and move things like this to that closure?

Reply:
Done in f223b981